tests/core/test_config_utils.py

```diff
@@ -126,3 +128,12 @@ def test_ptl_config(self):
         assert signatures_match
         assert cls_subset is None
         assert dataclass_subset is None
+
+    @pytest.mark.unit
+    def test_early_stopping_config(self,):
```

Comment:
@titu1994 I can't figure out what is wrong here.

Reply:
Seems to have worked out